### ISSUE TITLE
Hazard data: volcanic ash

### ISSUE THREAD

**Issue by Charlotte-Morgan** [inasafe/inasafe#3781]

**Comment by ivanbusthomi** [comment]
Hi @Charlotte-Morgan :

In the potential impact table on Volcanic ash Realtime report, there are descriptions of the impact on both buildings and roads, even though the unit is in cm.

![image](https://cloud.githubusercontent.com/assets/7351977/22682739/10991128-ed47-11e6-87be-b238de922c31.png)

Should we change it in the future?

**Comment by Charlotte-Morgan** [comment]
No, probably not.  This is something we can discuss with PVMBG soon.
For now the comments are for desktop applications. No change to realtime.


On 7 Feb 2017, at 7:03 PM, Ivan Busthomi <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi @Charlotte-Morgan<https://github.com/Charlotte-Morgan> :

In the potential impact table on Volcanic ash Realtime report, there are descriptions of the impact on both buildings and roads, even though the unit is in cm.

[image]<https://cloud.githubusercontent.com/assets/7351977/22682739/10991128-ed47-11e6-87be-b238de922c31.png>

Should we change it in the future?

-

**Comment by samnawi** [comment]
If I look at the default classification in the keyword wizard, It has same classification with the realtime.
![image](https://cloud.githubusercontent.com/assets/8871958/22820429/53b320b6-efa9-11e6-8b91-fab9714581b6.png)

And if we see the explanation in the realtime, it also mention the impact for building and road.

**Comment by Charlotte-Morgan** [comment]
@RikkiWeber - do you have any descriptions for volcanic ash hazard classes other than the one provided in the table above for realtime?
As the team note; while we say ash thickness is not relevant for the analysis of impact on buildings  or people, the descriptions seem to be more about the hazard class as a whole.

**Comment by RikkiWeber** [comment]
@Charlotte-Morgan - I think that we best put this on the list of things to discuss with BG next week. 
You are right that the table gives examples of what will be impacted by various ash thicknesses and the table is correct. I have provided a bit more detail on my understanding below, but let's leave it and make sure we are doing what BG actually wants as having three different measures will be more difficult.

Ash load/density for impact allow for more sophisticated analysis in the future, particularly if building information were available. It also allows for wet ash (raining during ashfall) to be more easily accounted for. 

For impact on people ash concentration at ground level is also more useful as it is possible that wind conditions mean that ash remains in the air and not necessarily directly related to the thickness on the ground.

**Comment by Charlotte-Morgan** [comment]
This was discussed with Ibu Estu adn Pak Nugi at PVMBG.
The advice is that ash thickness can be used as a surrogate for density and concentration, is easier for general users to understand and is thus relevant to all exposure types.
#3928 addresses some of this. more work is needed on thresholds for each exposure type but for now we will use the classes defined in Ash realtime.
@estukriswati @RikkiWeber @lucernae @ivanbusthomi
